pyprep/find_noisy_channels.py

```diff
@@ -68,6 +68,7 @@ def __init__(self, raw, do_detrend=True, random_state=None):
 
         # random_state
         self.random_state = check_random_state(random_state)
+        self.random_ch_picks = []
```

Comment:
I'd add a comment here

```suggestion
        self.random_ch_picks = []  # needed for ransac
```